docs/guide/blog/index.md

```diff
@@ -27,7 +27,7 @@ By completing this tutorial, you will learn about:
 This series of blog tutorials is based on a specific version of Ignite CLI, so to install Ignite CLI v0.20.0 use the following command:
 
 ```bash
-curl https://get.ignite.com/cli@v0.20.0! | bash
+curl https://get.ignite.com/cli! | bash
```

Comment:
oops, let's give a specific version and lets specify the latest version, can you make the same changes I suggested in https://github.com/ignite-hq/cli/pull/2413 ? thank you @alijnmerchant21